project_task_analytic_tag/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Project Task Analytic Tag
+=========================
+
+This module extends the functionality of project and mrp_repair to support to
```

Comment:
Isnot this module overlapping with https://github.com/OCA/project/tree/10.0/project_categ and https://github.com/OCA/project/tree/10.0/project_categ_issue?

I am not so much in favor of adding a new category system without clear value